cpp/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ elseif(CMAKE_CUDA_ARCHITECTURES STREQUAL "")
   set(CUDF_BUILD_FOR_DETECTED_ARCHS TRUE)
 endif()
 
-project(CUDF VERSION 0.19.0 LANGUAGES C CXX)
+project(CUDF VERSION 0.20.0 LANGUAGES C CXX)
```

Comment:
This one confuses me? Seems this should have been updated as part of the release as per this release script https://github.com/rapidsai/cudf/blob/branch-0.19/ci/release/update-version.sh#L53

Reply:
Updated `update-version.sh` with the new name